include/avif/avif.h

```diff
@@ -602,9 +578,26 @@ AVIF_API uint32_t avifRGBImagePixelSize(const avifRGBImage * rgb);
 AVIF_API void avifRGBImageAllocatePixels(avifRGBImage * rgb);
 AVIF_API void avifRGBImageFreePixels(avifRGBImage * rgb);
 
-// The main conversion functions
-AVIF_API avifResult avifImageRGBToYUV(avifImage * image, const avifRGBImage * rgb);
-AVIF_API avifResult avifImageYUVToRGB(const avifImage * image, avifRGBImage * rgb);
+// The main conversion options.
+typedef enum avifConversionFlag
+{
+    // Conversion between RGB and YUV samples:
+    AVIF_CONVERSION_AUTO = 0,                // prefers libyuv if available (default)
+    AVIF_CONVERSION_AVOID_LIBYUV = (1 << 0), // only picks built-in paths
+
+    // Conversion from RGB to YUV (ignored when converting from RGB to YUV), set at most one:
+    AVIF_CHROMA_DOWNSAMPLING_AVERAGE = (1 << 10), // only use the averaging filter (libyuv or built-in) (default)
+
+    // Conversion from YUV to RGB (ignored when converting from YUV to RGB), set at most one:
```

Comment:
In the parentheses, change "from YUV to RGB" to "from RGB to YUV".

These three flags specify the chroma upsampling filter, so the comment should start a new subsection for them, e.g.,

```
    // Conversion from YUV to RGB (ignored when converting from RGB to YUV)

    // Chroma upsampling filter. Set at most one:
```

We should say that they are ignored if the conversion does not need to upsample chroma (when input is YUV 4:4:4 or YUV 4:0:0). We shoudl also say what is the default chroma upsampling filter when none is specified.

Reply:
> In the parentheses, change "from YUV to RGB" to "from RGB to YUV".

Whoops.

> These three flags specify the chroma upsampling filter, so the comment should start a new subsection for them

Will there ever be another subsection? I find the `avifConversionFlag` enum difficult to read now.

> We should say that they are ignored if the conversion does not need to upsample chroma (when input is YUV 4:4:4 or YUV 4:0:0).

Done.

> We shoudl also say what is the default chroma upsampling filter when none is specified.

I did that through the `(default)` comments but I guess it was not visible or clear enough.